### ISSUE TITLE
Negative percents and vote counts

### ISSUE THREAD

**Issue by MatthewKosloski** [daryl/qstn#7]
I clicked "Random poll" and came across [this](http://qstn.co/q/OvfyLR7C) poll that appears to contain negative values.

**Comment by daryl** [comment]
Yeah there's a few like that, not sure if they're pre-update or some unknown bug exists.

I may just rewrite this early in the new year if I get time.
